manifests/06_service-catalog-apiserver.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: ServiceCatalogAPIServer
+metadata:
+  name: cluster
+spec:
+  logLevel: "Normal"
+  managementState: Removed
```

Comment:
i'm pretty sure that by doing this, the CVO will start stomping this field back to this value if it's edited (no one will be able to change it to anything else)

if you look at other operators, they create the CR resource programmatically, not via manifests/CVO.

Reply:
It has worked on all my tests, but now that you say this, I've always disabled the CVO to get my own operator deployed.  What you are saying makes a ton of sense, thanks Ben, I'll revisit.